Tests/Functional/Queue/DoctrineQueueTest.php

```diff
@@ -26,6 +26,6 @@ class DoctrineQueueTest extends AbstractQueueTest
      */
     protected function getQueue()
     {
-        return new DoctrineQueue('Test-queue', $this->queueSettings);
+        return new DoctrineQueue('test-queue', $this->queueSettings);
```

Comment:
That fixed the tests for mw, MariaDB would complain about a non-existing table, because of case differences between `flowpack_jobqueue_messages_Test-queue` and `flowpack_jobqueue_messages_test-queue`.

Not really a "proper" fix for that, but I was focused on something entirely different… 😎

Reply:
I'm still puzzled, that this failed for you since MariaDB should be case insensitive regarding table and column names – at least I _thought_ it was

Reply:
I looked at the docs and it wasn't as clear as I hoped it to be. 🤷

Reply:
quoting
https://dev.mysql.com/doc/refman/8.0/en/identifier-case-sensitivity.html

> Consequently, the case sensitivity of the underlying operating system plays a part in the case sensitivity of database, table, and trigger names. This means such names are not case-sensitive in Windows, but are case-sensitive in most varieties of Unix

Reply:
Ah, I was probably mislead by the fact that my MariaDB runs "on Linux" (in Docker), but uses the host filesystem for storage (on a Mac, case-insensitive). 🙄

Reply:
Maybe we should just strtolower it just to be on the safe side?

Reply:
Having it lowercase in the test code is fine, IMHO. Most people probably use lowercase dientifers for thir queues anyway…